packages/types/news/6254.feature

```diff
@@ -0,0 +1,3 @@
+Added optional 'path' parameter to querystringSearch functionality. This enables scoped searches, 
+allowing for more efficient and context-specific queries. Updated tests and schema to support 
```

Comment:
@askadityapandey lol LLM commit message pushed without checking

Reply:
I guess there was something in the contribution guidelines to do this. It has nothing to do with LLM. Lmao